packages/notation-image-verifier/notation-image-verifier.spec

```diff
@@ -0,0 +1,35 @@
+%global _cross_first_party 1
+%global workspace_name notation-image-verifier
+
+Name: %{_cross_os}%{workspace_name}
+Version: 0.1.0
+Release: 1%{?dist}
+Summary: Binary for containerd image verification plugin
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+BuildRequires: %{_cross_os}glibc-devel
```

Comment:
This package should probably require the notation CLI and perhaps the signer plugin also.